config/sync/views.view.health_conditions.yml

```diff
@@ -295,6 +297,10 @@ display:
       display_extenders: {  }
       path: services/health-conditions-a-z
       exposed_block: true
+      cache:
+        type: none
```

Comment:
Shouldn't we have caching by parameter/tag?

Reply:
Yeah not sure why this is set to none - wasn't something I altered in the view.  Will take a look